### DISCUSSION TITLE
.spz file support

### DISCUSSION THREAD

**Discussion by miranbrajsalate** [mkkellogg/GaussianSplats3D#356] [Ideas]
Hi @mkkellogg, hope you're doing well!

I was wondering if you've had plans to support the new .spz file format announced [here](https://www.linkedin.com/posts/joeludwin_3dgraphics-opensource-ar-ugcPost-7257109504318701569-xlvj)? It would be a huge improvement in many aspects, e.g. download/loading of the splats etc.

**Comment by mkkellogg** [comment]
I definitely would like to add support for this format, as well as the compression scheme mentioned [here](https://github.com/mkkellogg/GaussianSplats3D/discussions/309). It's just a matter of finding the time these days.

**Reply by miranbrajsalate** [reply]
Yeah, understandable. Would be awesome for those functionalities to find their way into the project eventually 🤞
